server/modules/collectImport/service/collectImport/metaImportJobs/nodeDefsImportJob.js

```diff
@@ -359,15 +370,12 @@ export default class NodeDefsImportJob extends Job {
       if (checkExpressionParser) {
         const collectExpr = checkExpressionParser(element)
         const messages = CollectSurvey.toLabels('message', defaultLanguage)(element)
-        const { condition } = CollectSurvey.getAttributes(element)
-
-        await this.addNodeDefImportIssue(
-          nodeDefUuid,
-          CollectImportReportItem.exprTypes.validationRules,
-          collectExpr,
-          condition,
-          messages,
-        )
+        const { if: condition, flag } = CollectSurvey.getAttributes(element)
+        const exprType =
+          flag === 'error'
+            ? CollectImportReportItem.exprTypes.validationRuleError
+            : CollectImportReportItem.exprTypes.validationRuleWarning
```

Comment:
use validation rule flag (severity) to determine expression type (validationRuleError | validationRuleWarning)